### PR TITLE
[Feature] Update vue-core and refactor Date models

### PR DESCRIPTION
**Pull request by Tanya-Amber-L**

### CODE REVIEW

File: src/models/date/calendarTime.transformer.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+export class CalendarTimeTransformer {
+  static fromDto(time: CalendarTimeDto): CalendarTime {
+    return time.toString() as CalendarTime
+  }
+
+  static fromNullableDto(date: CalendarTimeDto | null): CalendarTime | null {
+    return date === null ? null : this.fromDto(date)
+  }
+
+  static fromTime(time: string): CalendarTime {
```
Comment (maltsavkiryl) [inline src/models/date/calendarTime.transformer.ts:15]: hebben we deze nodig?

File: package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
     "@vueuse/core": "10.11.0",
     "@vueuse/router": "10.11.0",
     "@wisemen/oauth2-vue-client": "1.0.0",
```
Comment (maltsavkiryl) [inline package.json:30]: "@internationalized/date" mag weg

File: package.json [hunk 3]
```diff
@@ -28,7 +28,7 @@
...
-    "@wisemen/vue-core": "0.0.102",
+    "@wisemen/vue-core": "0.1.6",
     "axios": "1.7.3",
     "clsx": "2.1.1",
```
Comment (maltsavkiryl) [inline package.json:33]: ik zou even alle dependencies bumpen

File: src/components/layout/sidebar/AppSidebarMenuTooltipContent.vue [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<script setup lang="ts">
+import {
+  AppKeyboardShortcut,
+  AppText,
+} from '@wisemen/vue-core'
+import type { NavigationItemOption } from 'node_modules/@wisemen/vue-core/dist/types/navigationItem.type'
```
Comment (JeroenVanC) [inline src/components/layout/sidebar/AppSidebarMenuTooltipContent.vue:6]: gekke import?

**Review by JeroenVanC — APPROVED** [review]

**Pull request merged**